runtime/ftplugin/sudoers.vim

```diff
@@ -16,11 +16,11 @@ let b:undo_ftplugin = "setl com< cms< fo<"
 setlocal comments=:# commentstring=#\ %s formatoptions-=t formatoptions+=croql
 
 if has('unix') && executable('less')
-  if !has('gui_running')
+  if !has('gui_running') && !has('nvim')
     command -buffer -nargs=1 SudoersKeywordPrg
           \ silent exe '!' . 'LESS= MANPAGER="less --pattern=''\b' . <q-args> . '\b'' --hilite-search" man ' . 'sudoers' |
           \ redraw!
-  elseif has('terminal')
+  elseif executable(':terminal') == 2
```

Comment:
this returns 0 on Vim. did you mean `exists(':terminal')` ?

Reply:
Oh, typo. Thank you